requirements.txt

```diff
@@ -18,3 +18,7 @@ uvicorn==0.23.2
 uvloop==0.17.0
 watchfiles==0.19.0
 websockets==11.0.3
+bank-statement-reader-altara~=1.3.6
```

Comment:
Why are we having this as a dependency?

Reply:
I developed it as a package and uploaded to pip

Reply:
Let's add it to the repo(not as a package), our tech is getting too big

Reply:
Okay now